README.md

```diff
@@ -38,20 +38,7 @@ before running any gRPC code.
 
 ## Specifying a Project ID
 
-Most `gcloud-dotnet` libraries require a project ID. If you don't remember yours (or haven't created a project yet), navigate to the [Google Developers Console][google-developers-console] to view your project ID (or create a new project and get the ID then). Once done, there are multiple ways to specify the project ID for the client libraries to retrieve the value:
-
-1. When using `gcloud-dotnet` libraries from within Compute/App Engine, there's no need to specify a project ID.  It is automatically inferred from the production environment.
-2. When using `gcloud-dotnet` libraries elsewhere, you can do one of the following:
-    * Define the environment variable GCLOUD_PROJECT to be your desired project ID. For example:
-
-      ```
-      set GCLOUD_PROJECT=PROJECT_ID
-      ```
-    * If running locally for development/testing, set the project ID using the [Google Cloud SDK][google-cloud-sdk].  Download the SDK if you haven't already, and set the project ID by running the following in the command line:
-
-      ```
-      gcloud config set project PROJECT_ID
-      ```
+Most `gcloud-dotnet` libraries require a project ID. If you don't remember yours (or haven't created a project yet), navigate to the [Google Developers Console][google-developers-console] to view your project ID (or create a new project and get the ID then). Once done, record the value and make sure to pass it as a parameter to the methods that require it.
```

Comment:
I see that we mention it in the authentication... maybe this should all actually be in the requirements section. Basically, before you write any code, you need the bits, and a project with billing and the relevant APIs enabled.